### PR TITLE
Fixed removing empty orders too early.

### PR DESCRIPTION
**Pull request by marcovc**
Addresses #47.

### CODE REVIEW

File: src/token_pair_solver/solver.py [hunk 1]
```diff
@@ -367,9 +367,6 @@ def solve_token_pair_and_fee_token(
         token_pair, b_orders, s_orders, f_orders, xrate, b_buy_token_price, fee
     )
 
-    # Remove unfilled orders.
```
Comment (josojo) [inline src/token_pair_solver/solver.py:370]: What is the impact, if we are removing the unfilled orders? 

Probably, I don't understand it well enough, but I feel like it should stay there. Then also your test `or is_trivial(orders):` is more likely to return the right value.

Comment (marcovc) [reply src/token_pair_solver/solver.py:370]: In the PR before this one I added this statement to remove the orders with sell amount = 0 (empty orders). But this was being done before the final utility values were computed, which mean that they reflected the "touched orders" version of the objective, and not the "all orders" version we're optimizing for.

Comment (josojo) [reply src/token_pair_solver/solver.py:370]: But if you don't remove them, then your count in this function
```
def is_trivial(orders):
    return count_nr_exec_orders(orders) == 0
``` 
will no longer return 0, right?

Comment (marcovc) [reply src/token_pair_solver/solver.py:370]: It should, that function counts the number of executed orders, that is, orders for which buy_amount > 0.

Comment (marcovc) [reply src/token_pair_solver/solver.py:370]: To have a count of all orders it would suffice to do `len(orders)`.

Comment (josojo) [reply src/token_pair_solver/solver.py:370]: ups... sorry. Alright

Comment (marcovc) [reply src/token_pair_solver/solver.py:370]: No worries, always suspect! :)

**Pull request merged**